### ISSUE TITLE
Set "Fas group" to false for a group

### ISSUE THREAD

**Issue by rrotondo** [fedora-infra/freeipa-fas#150]
Hi, 
I'm trying to set "Fas group" to false but I don't find any option. I tried: 
```
ipa group-mod uiss --fasgroup
ipa: ERROR: no modifications to be performed
```
supposing it could work as a switch. 

Any hint?
Thank you in advance.

**Comment by hale177** [comment]
I also have this requirement. I want to disable  after enabling the fas group, but I can't find cli the relevant opts/ags

Forgot to say that the UI is not allowed to operate

**Comment by hale177** [comment]
@rrotondo  i resolve it   

`ipa  group-mod  xxx_group  --delattr=objectclass=fasgroup`

**Comment by abompard** [comment]
Yeah, that's the only way to do it at the moment. Thanks for posting it here.
